### DISCUSSION TITLE
Problem external image (capacitor/android)

### DISCUSSION THREAD

**Discussion by LucasMarengo** [quasarframework/quasar#10867] [CLI - Capacitor / Cordova modes]
Good morning, I am developing an app for Android with a capacitor.

But I'm having a problem, when building with capacitor for Android, the external url images are not being displayed
example: <q-img src="https://cdn.quasar.dev/logo-v2/svg/logo-vertical.svg"></q-img>

**Comment by ontwikkelfabriek** [comment]
@LucasMarengo 

Connect the device to your computer and run Android Studio. Open the device logger, select your device and you'll see the log output. Somewhere in that output you'll find the reason why.
